cards/README.md

```diff
@@ -1,53 +1,54 @@
 
 ## Add Card
 
-To add a new card, go to the **API Playground** on the dashboard, choose **Add Card** from the left menu endpoints. The method will be set to POST and the endpoint to **_/cards_**. An editable sample JSON object like the one displayed below will be used to create the card. In order to add a card from the Playground, you’ll need to use one of the available testing card numbers displayed below. Also, provide an expiry date in the future, `countryCode` and `programId` you want to link this card to. You must set `termsOfUse` to `true` to define that the user agreed to the terms of use and opt-in. 
+### Testing Card Addition: Playground
+To test the addition of a new card to a program, go to the [**API Playground**](https://dashboard.fidel.uk/playground), and choose **Add Card** from the left menu endpoints.  The method is set to POST and the endpoint to **_/cards_**.  When you choose the program that you would like to connect the card to from the dropdown, the POST url will change to show the program ID.  In the edit field of the playground is the JSON object that will be used to add the card. To add a card, use one of the e available testing card numbers displayed below, an expiry date in the future, the three letter `countryCode`. You must set `termsOfUse` to `true` to define that the user agreed to the terms of use and opt-in. 
 
-You can use the dropdown menus to automatically set the ids in the request body or copy them to use in your application code without leaving the playground.
-
-<div class="info-box">
-    <small>Important note</small><br/>
-    To use the <strong>Create Card</strong> endpoint you must use the test public key. This only applies to this endpoint. Using the <strong>Create Card</strong> endpoint on live environment requires your company to be PCI Compliant. If you want to use it instead of the SDKs, please contact us at developer@fidel.uk.
-</div>
+>
+>    <small>Important note</small><br/>
+>    To use the <strong>Create Card</strong> endpoint, you must use the test public key. Using the <strong>Create Card</strong> API endpoint on live environment requires your company to be PCI Compliant (and will use the secret key. If you want to use it instead of the SDKs, please contact us at developer@fidel.uk.
 
 ##### Create sample cards in test mode using the API Playground.
 
 ![Create card](https://docs.fidel.uk/assets/images/create-card.png "Create card")
 
-If the card is successfully linked, the newly created card object is returned in JSON and displayed in the response body box. You can also use the [Web SDK](/web-sdk) to create cards in test environment using your SDK test key. If an error occurs on card creation, you receive the error message in the HTTP response body.
+When the card is successfully linked, the newly created card object is returned in JSON and displayed in the response body box. IF the card lining fails, the failure object will be delivered instead.  
+### Testing Card Addition: Web SDK
```

Comment:
change title to `Testing card-linking using the Web SDK`

Reply:
updating line 12 for playground to match